README.md

```diff
@@ -21,6 +21,25 @@ The `push-image` action builds a Docker image and pushes it to an appropriate Go
     labels: project=myproject
 ```
 
+#### How to push images to ECR
+
+to push images to ECR you need to provide an ECR `dockerRegistry` and setup few environment variable. For your image to be deployable via slipstream it is mandatory to set the `release` field to `true` otherwise your image will be considered as a development image.
+
+```yaml
+- name: Build and push Docker image
+  uses: BrandwatchLtd/slipstream-actions/push-image@main
+  id: push-to-slipstream
+  with:
+    service: slipstream-pipeline-visualiser
+    labels: project=appinfra
+    dockerRegistry: <your_ecr_docker_registry>
+    release: true #for image to be deployable via slipstream
+  env:
+    AWS_ACCESS_KEY_ID: ${{ secrets.ARTIFACTS_PROD_AWS_ACCESS_KEY }}
+    AWS_SECRET_ACCESS_KEY: ${{ secrets.ARTIFACTS_PROD_AWS_SECRET_KEY }}
```

Comment:
Maybe it's worth mentioning that `ARTIFACTS_PROD_AWS_ACCESS_KEY` and `ARTIFACTS_PROD_AWS_SECRET_KEY` are org secrets provided by us, so there is no need to set them on a repo level (unless you want to use a different account)